.github/workflows/ipc_fuzzer.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+# This is NOT how to properly compile the fuzzer, this is only to detect
+# earlier and faster problems like
```

Comment:
Isn't this the proper way to compile the fuzzer now?

Reply:
Unfortunately `mv` also doesn't update comments